docs/examples/kotest/build.gradle

```diff
@@ -0,0 +1,39 @@
+plugins {
```

Comment:
IMO this should go to the module's tests

Reply:
Removed